examples/step-69/doc/results.dox

```diff
@@ -0,0 +1,148 @@
+    #########                                  #########
+    #########     Cycle  007553  (100.0%)      #########
+    #########      at time t = 3.99984036      #########
+    #########                                  #########
+    ####################################################
+
+TimeLoop<dim>::output(t = 4.00038, checkpoint = 1)
+
++------------------------------------------------------------------------+------------+------------+
+| Total CPU time elapsed since start                                     |       357s |            |
+|                                                                        |            |            |
+| Section                                                    | no. calls |  CPU time  | % of total |
++------------------------------------------------------------+-----------+------------+------------+
+| discretization - setup                                     |         1 |     0.113s |         0% |
+| offline_data - assemble lumped mass matrix, and c_ij       |         1 |     0.167s |         0% |
+| offline_data - compute |c_ij|, and n_ij                    |         1 |   0.00255s |         0% |
+| offline_data - create sparsity pattern and set up matrices |         1 |    0.0224s |         0% |
+| offline_data - distribute dofs                             |         1 |    0.0617s |         0% |
+| offline_data - fix slip boundary c_ij                      |         1 |    0.0329s |         0% |
+| schlieren_postprocessor - compute schlieren plot           |       201 |     0.811s |      0.23% |
+| schlieren_postprocessor - prepare scratch space            |         1 |   7.6e-05s |         0% |
+| time_loop - setup scratch space                            |         1 |     0.127s |         0% |
+| time_loop - stalled output                                 |       200 |  0.000685s |         0% |
+| time_step - 1 compute d_ij                                 |      7553 |       240s |        67% |
+| time_step - 2 compute d_ii, and tau_max                    |      7553 |      11.5s |       3.2% |
+| time_step - 3 perform update                               |      7553 |       101s |        28% |
+| time_step - 4 fix boundary states                          |      7553 |     0.724s |       0.2% |
+| time_step - prepare scratch space                          |         1 |   0.00245s |         0% |
++------------------------------------------------------------+-----------+------------+------------+
+@endverbatim
+
+One thing that becomes evident is the fact that the program spends two
+thirds of the execution time computing the graph viscosity d_ij and about a
+third of the execution time in performing the update, where computing the
+flux $f(U)$ is the expensive operation. The preset default resolution is
+about 37k gridpoints, which amounts to about 148k spatial degrees of
+freedom in 2D. An animated schlieren plot of the solution looks as follows:
```

Comment:
Maybe link to https://en.wikipedia.org/wiki/Schlieren ?

Reply:
Added link nearby the description if the class SchlierenPostprocessor.